JSTests/wasm/stress/validate-function-signature-in-section-parser.js

```diff
@@ -0,0 +1,23 @@
+try {
+    let v28 = 96;
+    const v41 = new BigInt64Array(8);
+    v41.__proto__;
+    const o43 = {
+    };
+    Math.tanh(1000000.0);
+    ~1000000.0;
+    const o64 = {
+    };
+    const o66 = {
+    };
+    o66.g = 1;
+    const o67 = {
+    };
+    const o85 = {
+    };
+    [BigInt64Array,48];
+    v28--;
+    const v90 = new Uint8Array([0,97,115,109,1,0,0,0,1,21,5,96,1,127,0,96,1,127,0,96,1,127,0,96,1,127,0,v28,1,127,0,2,39,5,3,102,111,111,1,48,0,0,1,48,3,102,111,111,0,1,1,48,1,48,0,2,2,49,56,2,45,51,0,3,2,45,51,2,49,56,0,4]);
```

Comment:
This test should be more simpler? For example, I would like the following error:

```js
shouldThrow(function () {
    const bin = new Uint8Array([0,97,115,109,1,0,0,0,1,21,5,96,1,127,0,96,1,127,0,96,1,127,0,96,1,127,0,95,1,127,0,2,39,5,3,102,111,111,1,48,0,0,1,48,3,102,111,111,0,1,1,48,1,48,0,2,2,49,56,2,45,51,0,3,2,45,51,2,49,56,0,4]);
    new WebAssembly.Module(bin);
}, WebAssembly.CompileError);
```

Reply:
Done.